scripts/update-backend-sdk.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ "$1" == "--help" ]; then
+  echo "Usage: $0 [vX.Y.Z]"
+  echo "(defaults to latest version if not specified)"
+  exit 0
+fi
+
+SDK_TARGET="${1:-latest}"
+
+# Keep track of base directory
+base=$(pwd)
+
+# Find go.mod files (backend plugins)
+files=$(find examples -type f -name "go.mod")
+
+# Iterate over each file and run `go get` to upgrade the grafana-plugin-sdk-go dependency
+for file in $files; do
+  echo "Upgrading grafana-plugin-sdk-go to $SDK_TARGET in $file"
+
+  cd $(dirname "$file")
```

Comment:
bash trick. use `pushd $(dirname "$file") || exit 1` to enter a dir and `popd || exit 1` to go back to wherever  the shell was before the push, that way you don't need to keep track of the original CWD

Reply:
Awesome, I applied the suggestion, thank you! :pray: